src/coreclr/src/vm/interoplibinterface.h

```diff
@@ -35,7 +35,7 @@ class ComWrappersNative
         _Inout_ QCall::ObjectHandleOnStack retValue);
 
 public: // Lifetime management for COM Wrappers
-    static void DestroyManagedObjectComWrapper(_In_ void* wrapper);
+    static void DestroyManagedObjectComWrapper(_In_ void* wrapper, void *unused);
```

Comment:
Why does this function need an unused argument?

Reply:
The DAC needs to get a list of all ComWrappers for the object, so I refactored `ClearManagedObjectComWrappers` and created the `IterateComWrappers` method that just iterates so the DAC can call it. The unused parameter here is a side effect of trying to share code as much as possible, the DAC needs to be able to pass in an array to accumulate the ComWrappers and have it passed to the callback. There were two instances of code that called `ClearManagedObjectComWrappers` that had to be updated when the function signature was updated.

Reply:
Okay. Can you please label is with the following SAL - `_Out_opt_`. The name is good by the way - appreciate it.